packages/itwin/grouping-mapping-widget/src/widget/components/CustomCalculationAction.tsx

```diff
@@ -225,6 +226,13 @@ export const CustomCalculationAction = ({
               validator.showMessageFor("name");
             }}
           />
+          <Alert
+            type='informational'
+            clickableText='Click here.'
```

Comment:
What does `clickableText` do?

Reply:
I used a feature from the iTwin UI bank. It looks like this https://github.com/iTwin/viewer-components-react/assets/22688247/17c5eab3-1560-4600-8cc8-561ccd3880c6